fv3core/stencils/d_sw.py

```diff
@@ -426,51 +392,42 @@ def max_d2_bg1():
     return max(spec.namelist.d2_bg, spec.namelist.d2_bg_k2)
 
 
-def get_column_namelist():
-    ks = [k[0] for k in k_bounds()]
-    col = {}
-    for ki in ks:
-        col[ki] = column_namelist_options(ki)
-    return col
-
-
-def get_single_column(key):
-    col = []
-    for k in range(0, grid().npz):
-        col.append(column_namelist_options(k)[key])
+def get_single_column(column_namelist, key):
```

Comment:
It would be great to record here why you would want to get a single column of something, and what a "single column" is.

Reply:
or remove them now :). it was to pass the parts of the column namelist we need into updatedzd, but now will change just to call the same get_column_namelist function -- silly to do, but when we have objects it can be done in init, and when we refactor away the need for column namelist it will go away